src/components/Editor/index.js

```diff
@@ -174,12 +175,26 @@ class Editor extends Component {
       const plain = this.getPlain()
 
       this.recordChange(plain, this.selection)
-      this.updateContent(plain)
+      !this.compositing && this.updateContent(plain);
```

Comment:
Can you replace this with an if statement (including a body I.e. curly braces) please?

Also let’s move it into updateContent itself, since this method is a little bloated already

Reply:
New commit [d88b23](https://github.com/FormidableLabs/react-live/pull/46/commits/d88b23b4bc5acb793896b9efeab6fb6891d89ea6) works for it.